packages/backend-core/src/cache/base/index.ts

```diff
@@ -23,6 +23,18 @@ export default class BaseCache {
     return client.keys(pattern)
```

Comment:
This file is exposing some of the redis utils into our cache. I ended up not using it, but it is worth keeping